src/tools/auth0/handlers/index.ts

```diff
@@ -20,7 +20,6 @@ import * as guardianPhoneFactorMessageTypes from './guardianPhoneFactorMessageTy
 import * as roles from './roles';
 import * as branding from './branding';
 import * as prompts from './prompts';
-import * as migrations from './migrations';
```

Comment:
Migrations not part of the public spec. Need to understand if this was an intentional deprecation or if it is even necessary.